### ISSUE TITLE
Semantics

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/Collecting#96]
- Under "Form Information" and when editing a form, the "Prompts" heading should be `<h2>`.
- Also under "Form Information": "Label", "Anonymity Type", and "Item Set" are tagged as `<h4>`. Perhaps the "Form Information" tab should be `<h2>`, while the former property categories would be `<h3>`? (In that case, "Collected Items" would also receive an `<h2>` tag)
